broker/thinBroker.go

```diff
@@ -2837,7 +2837,11 @@ func (tb *ThinBroker) updateAttributes(elem map[string]interface{}, eid string)
 	tb.ldEntities[eid] = entityMap
 
 	// update registration on discovery
-	tb.registerLDContextElement(entityMap)
```

Comment:
[minor] Add proper comment this request is not forwarded to discovery.

Reply:
done